apps/omg_api/lib/api.ex

```diff
@@ -23,13 +23,13 @@ defmodule OMG.API do
   alias OMG.API.{Block, Core, FeeChecker, FreshBlocks, State}
   use OMG.API.LoggerExt
 
-  @type submit_error() :: Core.recover_tx_error() | FeeChecker.error() | State.exec_error()
+  @type submit_error() :: Core.recover_tx_error() | State.exec_error()
 
   @spec submit(transaction :: binary) ::
           {:ok, %{txhash: <<_::768>>, blknum: pos_integer, txindex: non_neg_integer}} | {:error, submit_error()}
   def submit(transaction) do
     with {:ok, recovered_tx} <- Core.recover_tx(transaction),
-         {:ok, fees} <- FeeChecker.transaction_fees(recovered_tx),
+         {:ok, fees} <- FeeChecker.transaction_fees(),
```

Comment:
let's refresh the name of `FeeChecker` - the "Checker" part of the name has become completely outdated, it doesn't check anything anymore.

We can even make the full leap into the "newer" naming scheme:
`FeeChecker` -> `FeesServer` and `FeeChecker.Core` -> `Fees`

Reply:
Wow, I unaware of new naming. Will do

Reply:
we didn't really put a name to the "new naming" yet, but I think we were generally in favor of dicthing the `.Core` appendices